### DISCUSSION TITLE
JPEG XL support?

### DISCUSSION THREAD

**Discussion by mn7216** [chaiNNer-org/chaiNNer#2981] [Q&A]
Has JPEG XL support been considered? It's a pretty nice format and might go a long way with its efficiency both lossless-wise and lossy-wise

**Comment by joeyballentine** [comment]
If there's a good python library for reading it, we could add support fairly easily. Just gotta look into it

**Reply by WerctFourth** [reply]
> If there's a good python library for reading it, we could add support fairly easily. Just gotta look into it

PyVIPS? It may be better overall than opencv for saving/loading images
